[DOCS] New page: Detection & Response dashboard (#2085)
elastic/security-docs: docs/dashboards/detection-response-dashboard.asciidoc

[[detection-response-dashboard]]
= Detection & Response dashboard

> The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts.

Reviewer: ```suggestion
The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts.
```
Re: the deletion I suggested in the last line, even though it does highlight the most frequent and severe alerts if there are any, since it enables monitoring of all alerts I think this would be more accurate.

[role="screenshot"]
image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]

Interact with various dashboard elements:

> * Use the date and time picker in the upper-right to specify a timeframe for displaying information on the dashboard. 

Reviewer 2: ```suggestion
* Use the date and time picker in the upper-right to specify a time frame for displaying information on the dashboard. 
```

Reviewer 2: I've noticed we use `time range` more frequently when talking about the date and time range selected on the time picker but `time frame` works as well.

Author: I like "time range"!

Author: ```suggestion
* Use the date and time picker in the upper-right to specify a time range for displaying information on the dashboard. 
```

Author: I like "time range"!
```suggestion
* Use the date and time picker in the upper-right to specify a time range for displaying information on the dashboard. 
```

> * In sections that list alert counts, click the number to investigate the alerts in Timeline.

Reviewer: ```suggestion
* In sections that list alert counts, click a number to investigate those alerts in Timeline.
```

> * Click the names of detection rules, cases, hosts, and users to open their details pages.

Reviewer: ```suggestion
* Click the name of a detection rule, case, host, or user to view more details.
```

Reviewer: Since you can only click one at a time

Author: I want to convey that clicking a name opens a new page and navigates away from the dashboard, vs. just showing more details which could imply a less disruptive tooltip or flyout. Might be overthinking how much nuance readers pick up on, but gotta try at least. 😅 Also we seem to use plural for things like "rule details page" so this follows that pattern. Maybe something like:
```suggestion
* Click the name of a detection rule, case, host, or users to open its details page.
```

Reviewer: ^^ 💯

The following sections are included:

[width="100%",cols="s,"]
|==============================================

|Alerts

> |The total number of detection alerts generated in the time range, organized by status and severity. Select *View alerts* to open the Alerts page.

Author: ```suggestion
|The total number of detection alerts generated within the time range, organized by status and severity. Select *View alerts* to open the Alerts page.
```

> |The total number of detection alerts generated in the timeframe, organized by status and severity. Select *View alerts* to open the Alerts page.

Reviewer 2: ```suggestion
|The total number of detection alerts generated in the time frame, organized by status and severity. Select *View alerts* to open the Alerts page.
```

Author: ```suggestion
|The total number of detection alerts generated in the time range, organized by status and severity. Select *View alerts* to open the Alerts page.
```

|Cases

> |The total number of cases created in the time range, organized by status. Select *View cases* to open the Cases page.

Reviewer 3: ```suggestion
|The total number of cases created within the time range, organized by status. Select *View cases* to open the Cases page.
```

> |The total number of cases created in the timeframe, organized by status. Select *View cases* to open the Cases page.

Reviewer 2: ```suggestion
|The total number of cases created in the time frame, organized by status. Select *View cases* to open the Cases page.
```

Author: ```suggestion
|The total number of cases created in the time range, organized by status. Select *View cases* to open the Cases page.
```

|Open alerts by rule

> |The top four detection rules with alerts in the *Open* status, organized by the severity and number of alerts for each rule. Select *View all open alerts* to open the Alerts page.

Reviewer: ```suggestion
|The top four detection rules with open alerts, organized by the severity and number of alerts for each rule. Select *View all open alerts* to open the Alerts page.
```
I could be off base with removing the bold on "open", but I think we usually use bold on things users click, and here I think open is just an adjective.

|Recently created cases
|The four most recently created cases. Select *View recent cases* to open the Cases page.

|Hosts by alert severity

> |The hosts generating detection alerts in the time range, organized by the severity and number of alerts. Shows up to 100 hosts.

Reviewer 3: ```suggestion
|The hosts generating detection alerts within the time range, organized by the severity and number of alerts. Shows up to 100 hosts.
```

> |The hosts generating detection alerts in the timeframe, organized by the severity and number of alerts. Maximum 100 hosts listed.

Reviewer: ```suggestion
|The hosts generating detection alerts in the timeframe, organized by the severity and number of alerts. Shows up to 100 hosts.
```

Reviewer: If you decide to go with this, it applies to the next item as well.

Author: ```suggestion
|The hosts generating detection alerts in the time range, organized by the severity and number of alerts. Shows up to 100 hosts.
```

> |The hosts generating detection alerts in the timeframe, organized by the severity and number of alerts. Maximum 100 hosts listed.

Author: ```suggestion
|The hosts generating detection alerts in the time range, organized by the severity and number of alerts. Maximum of 100 hosts listed.
```

|Users by alert severity

> |The users generating detection alerts in the time range, organized by the severity and number of alerts. Shows up to 100 users.

Reviewer 3: ```suggestion
|The users generating detection alerts within the time range, organized by the severity and number of alerts. Shows up to 100 users.
```

> |The users generating detection alerts in the timeframe, organized by the severity and number of alerts. Maximum 100 users listed.

Reviewer 2: ```suggestion
|The users generating detection alerts in the timeframe, organized by the severity and number of alerts. Maximum of 100 users listed.
```

Author: ```suggestion
|The users generating detection alerts in the time range, organized by the severity and number of alerts. Shows up to 100 users.
```
